compiler/optimizer/Inliner.cpp

```diff
@@ -346,6 +346,14 @@ TR_InlinerBase::setInlineThresholds(TR::ResolvedMethodSymbol *callerSymbol)
    if (e)
       _nodeCountThreshold = atoi(e);
 
+   if (comp()->getOption(TR_EnableOSR))
+      {
+      static const char *f = feGetEnv("TR_OSRNodeCountThreshold");
+      if (f)
+         _nodeCountThreshold = atoi(f);
+      else
+         _nodeCountThreshold *= 2;
```

Comment:
How accurate is this node count estimate? Does OSR introduce twice the number of trees on average?